extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -39,17 +39,17 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
     onGeoJsonParsed: OnGeoJsonParsed
   ) : this(builder) {
     rawGeoJson?.let {
-      ignoreParsedGeoJsonRegistry[sourceId] = false
+      updateParsedGeoJsonRegistry.add(sourceId + it.hashCode())
```

Comment:
maybe save `sourceId + it.hashCode()` to some variable and not calculate it twice

same below